bin/straxer

```diff
@@ -214,15 +259,25 @@ def main(args):
         print(msg, flush=True)
 
     logging.info(f"\nStraxer is done! "
-          f"We took {time.time() - clock_start:.1f} seconds, "
-          f"peak RAM usage was around {peak_ram:.1f} MB.")
+                 f"We took {time.time() - clock_start:.1f} seconds, "
+                 f"peak RAM usage was around {peak_ram:.1f} MB.")
+
+
+def register_to_context(st, module: str):
+    if not os.path.exists(module):
+        raise FileNotFoundError(f'No such file {module}')
+    assert module.endswith('.py'), "only py files please!"
+    folder, file = os.path.split(module)
+    sys.path.append(folder)
+    to_register = importlib.import_module(os.path.splitext(file)[0])
+    st.register_all(to_register)
+    logging.info(f'Successfully registered {file}. Printing plugins')
```

Comment:
This is a pretty long print-out in the beginning we could reduce to just show the in addition registered once.

Reply:
Yeah I thought about that but it would require more work since you might also overwrite some plugins. I would rather be a bit more verbose than not know which plugins I use.